torch/distributions/kl.py

```diff
@@ -439,6 +440,9 @@ def _kl_normal_gumbel(p, q):
 
 # TODO: Add Normal-Laplace KL Divergence
 
+@register_kl(Poisson, Poisson)
```

Comment:
Extremely minor nit: Could you position this below `Pareto`?

Reply:
Yep sure.